docs/eck-attributes.asciidoc

```diff
@@ -1,5 +1,5 @@
-:eck_version: 1.4.0
+:eck_version: 1.6.0
```

Comment:
My bad, I forgot to update this during the last release.
This version must be the last existing release because it's used in the 'master' doc: https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-deploy-eck.html. https://download.elastic.co/downloads/eck/1.6.0/all-in-one.yaml does not exist yet.
```suggestion
:eck_version: 1.5.0
```